src/main/java/lotto/view/OutputView.java

```diff
@@ -1,50 +1,77 @@
 package lotto.view;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import lotto.model.winning.WinningResultResponse;
 import lotto.model.winning.WinningResultResponses;
+import lotto.util.NumberFormatter;
 
 public class OutputView {
 
-    private static final String WINNING_RESULT_FORMAT = "%d개 일치 (%d원)- %d개";
-    private static final String WINNING_SECOND_RESULT_FORMAT = "%d개, 보너스 볼 일치(%d원)- %d개";
+    public void printChangeAmount(final int changeAmount) {
+        System.out.printf("로또 구매 후 남은 잔돈은 %s원 입니다." + System.lineSeparator(), getFormattedMoney(changeAmount));
```

Comment:
```suggestion
        System.out.printf("로또 구매 후 남은 잔돈은 %s원 입니다.", getFormattedMoney(changeAmount));
        System.out.println();
```
`+ System.lineSeparator()`의 사용은 문자열 연산과 더불어 가독성을 저하하는 것으로 보이는데요.
개행만을 위한 `System.out.println()` 사용은 어떨까 싶어요.